### ISSUE TITLE
how did you train nasnet_large?

### ISSUE THREAD

**Issue by qingzhouzhen** [Cadene/pretrained-models.pytorch#24]
```
 Notes for training large NASNet model on ImageNet<br>
 batch size (per replica): 16
 learning rate: 0.015 * 100
 learning rate decay factor: 0.97
 num epochs per decay: 2.4
 sync sgd with 100 replicas
 auxiliary head loss weighting: 0.4
 label smoothing: 0.1
 clip global norm of all gradients by 10
```

I found this in nasnet.py in Tensorflow, but it need 100GPUs, I am going to use batch parallel, and by the way, it said it has a `# auxiliary head loss weighting: 0.4`, and in TF'code, it write like this
```   
  #############################
      # Specify the loss function #
  #############################
  if 'AuxLogits' in end_points:
      slim.losses.softmax_cross_entropy(
            end_points['AuxLogits'], labels,
            label_smoothing=FLAGS.label_smoothing, weights=0.4,
            scope='aux_loss')
      slim.losses.softmax_cross_entropy(
          logits, labels, label_smoothing=FLAGS.label_smoothing, weights=1.0)
      return end_points
```
how did you deal with this AuxLogits? use 100 GPUs like google? I know batch parelle which can simulate large batch size, would it affect accuracy? I change your code to MXNET, but can only arrive 8 in K80,
@Cadene

**Comment by Cadene** [comment]
@qingzhouzhen I did not train NASNet-A-Large, I ported it from the tensorflow slim repo (pretrained model). 

You can find how I did it there: https://github.com/Cadene/tensorflow-model-zoo.torch/tree/master/nasnet
